presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environments.java

```diff
@@ -109,11 +109,21 @@ public static void pruneNetworks()
 
     public static String nameForClass(Class<? extends EnvironmentProvider> clazz)
     {
-        return CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_HYPHEN, clazz.getSimpleName());
+        return canonicalName(clazz);
     }
 
     public static String nameForConfigClass(Class<? extends EnvironmentConfig> clazz)
     {
-        return CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_HYPHEN, clazz.getSimpleName());
+        return canonicalName(clazz);
+    }
+
+    private static String canonicalName(Class<?> clazz)
+    {
+        return canonicalName(clazz.getSimpleName());
+    }
+
+    public static String canonicalName(String name)
```

Comment:
```
/**
 * Converts camel case name to hyphenated. Returns input if the name is already hyphenated.
 */
```

-- the javadoc is important, as we do not commit ourselves to supporting anything beyond lowercase hyphenaed and CamelCase forms, and we should capture somewhere.